doc/user/data/sql_funcs.yml

```diff
@@ -307,6 +307,13 @@
   - signature: 'constant_time_eq(a: text, b: text) -> bool'
     description: Returns `true` if the strings are identical, otherwise returns `false`. The implementation mitigates timing attacks by making a best-effort attempt to execute in constant time if the strings have the same length, regardless of their contents.
 
+  - signature: 'initcap(a: text) -> text'
+    description: >-
+      Returns `a` with the first character of every word in upper case and all
+      other characters in lower case. Words are separated by non-alphanumeric
+      characters.
+    version-added: v0.97
```

Comment:
what's this?

Reply:
We recently introduced special syntax to handle not-yet-released feature documentation. See #25156 for context.

Reply:
And there's docs on what this does here: https://github.com/MaterializeInc/materialize/blob/main/doc/user/README.md#adding-documentation-for-new-features